modules/router/package.json

```diff
@@ -27,6 +27,7 @@
     "graphql-parse-resolve-info": "^4.5.0",
     "graphql-type-json": "^0.3.1",
     "helmet": "^5.1.0",
+    "http-proxy-middleware": "^2.0.6",
```

Comment:
remove proxy middleware from router dependencies, it's only a hermes dependency.